src/pages/Login/Login.tsx

```diff
@@ -78,23 +78,38 @@ const signInWithGoogle = async (firebase: Firebase): Promise<string | undefined>
 };
 
 export const Login: FunctionComponent = () => {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
   const firebase = useFirebase();
   const [signIn, { data }] = useMutation<SignInResponse>(SIGN_IN);
 
   if (!firebase) {
     return null;
   }
 
+  const handleBEConnection = async (idToken: string): Promise<void> => {
```

Comment:
handleLogin and handleBEConnection can be outside this function, right?

Reply:
Is it worth? handleBEConnection would require to pass `signIn` mutation in parameters. And handleLogin would need to get firebase.